parts/dcosagentvars.t

```diff
@@ -26,4 +26,5 @@
     "{{.Name}}LbIPConfigID": "[concat(variables('{{.Name}}LbID'),'/frontendIPConfigurations/', variables('{{.Name}}LbIPConfigName'))]", 
     "{{.Name}}LbIPConfigName": "[concat(variables('orchestratorName'), '-{{.Name}}-', variables('nameSuffix'))]", 
     "{{.Name}}LbName": "[concat(variables('orchestratorName'), '-{{.Name}}-', variables('nameSuffix'))]",
-{{end}}
\ No newline at end of file
+{{end}}
+    "dcosBootstrapURL": "[parameters('dcosBootstrapURL')]", 
```

Comment:
This parameter is in both agentvar and mastervar. I also created a new dcosparam.t to set the default value of the parameter. The value of this parameter is determined by the version of the DCOS specified in the cluster definition json. For example, if DCOS1.8.7 is specified, download url will be generated and set to the parameter based on the location of the cloud.